pennylane_qiskit/qiskit_device.py

```diff
@@ -141,6 +142,15 @@ class QiskitDevice(QubitDevice, abc.ABC):
     _eigs = {}
 
     def __init__(self, wires, provider, backend, shots=1024, **kwargs):
+
+        if qiskit.__version__ in ["0.46.0", "1.0.0"]:
```

Comment:
so we're allowing installation with the new qiskit version only for conversion purposes because they won't be able to make a device right?

also, if the goal here is "must be less than 0.46.0", you might want something more general, like this:
```pycon
>>> from semantic_version import Version
>>> max_ver = Version("0.46.0")
# here you'd pass `qiskit.__version__` into `Version(...)` and compare
>>> Version("0.45.9") < max_ver, Version("1.1.0") < max_ver
(True, False)
```

Reply:
+1. We should ensure only the version less than `0.46/1.0` should be present. 
As a side note - Qiskit won't allow installing `1.0` along with `0.46`, in case that would help with version checking.

Reply:
done!

Reply:
Hey there! By passing this error message and using Qiskit 1.0.1, I'm able to create a device perfectly fine. Is there something else I'm missing as to why the Qiskit device functionality is technically unsupported in 0.35?

Reply:
Hi @Josh-H-555! Thanks for asking, this is a good point. Qiskit 0.46/1.0 came out a bit close to our scheduled release. We didn't have enough time check everything we would need to in order to guarantee everything was working. We did some of the groundwork for compatibility, so some areas of the code base surrounding the devices will work as expected, but others won't. 

Rather than recommend something we weren't confident in, we opted to impose a version requirement until we can go through things more thoroughly and fix the remaining incompatibilities. We're planning to have a fix for more comprehensive 1.0 compatibility in the coming weeks!

Reply:
I see, that makes sense. Thanks for the quick reply! I look forward to the upcoming updates. :)